ts/IterableInboxSwipeableRow.tsx

```diff
@@ -67,7 +69,7 @@ const IterableInboxSwipeableRow = ({
          toValue: {x, y: 0},
          duration: FORCING_DURATION,
          useNativeDriver: false   
-      })//.start(() => deleteMessage(message.inAppMessage.messageId))
+      }).start(() => deleteMessage(rowViewModel.inAppMessage.messageId, index))
```

Comment:
I couldn't find a good place to put this, but line 91 is also throwing an error for me that we should be able to resolve.

Reply:
Also noted here, we shouldn't need both the `messageId` and the `index` for deletion.

Reply:
I will resolve that error and get rid of index in the delete item call.